docker-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+
+# TODO - generate this list of commands at docker-build-time
```

Comment:
Yeah - I wanted to discuss with @mholt first - not sure what the best way to do this would be, but my first thought was that maybe the `caddy` binary could have an option to spit these out on the commandline (like `--help` but formatted for this purpose).